controllers/onload_controller_test.go

```diff
@@ -165,34 +245,72 @@ var _ = Describe("Testing using mocked client", func() {
 				Times(1)
 
 			// Listing pods
-			mockClient.EXPECT().
+			listPodsCalls := mockClient.EXPECT().
 				List(gomock.Any(), &corev1.PodList{}, gomock.Any()).
 				SetArg(1, pods).
 				Return(nil).
-				Times(len(nodes.Items)).After(listNodesCall)
+				Times(len(nodes.Items)).
+				After(listNodesCall)
 
 			// Patching nodes to add the label
 			mockClient.EXPECT().
 				Patch(gomock.Any(), &nodes.Items[0], gomock.Any()).
 				Return(nil).
+				Times(1).
+				After(listPodsCalls)
+
+			Expect(r.addKmmLabelsToNodes(ctx, &onload)).Should(Equal(&ctrl.Result{Requeue: true}))
+		})
+
+		It("should label Nodes with Onload and SFC kmm labels", func() {
+			onload.Spec.Onload.KernelMappings[0].SFC = &onloadv1alpha1.SFCSpec{}
+
+			pods := corev1.PodList{}
+
+			// Listing nodes
+			listNodesCall := mockClient.EXPECT().
+				List(gomock.Any(), &corev1.NodeList{}, gomock.Any()).
+				SetArg(1, nodes).
+				Return(nil).
 				Times(1)
 
+			// This can be either "list nodes" or "patch node"
+			previousCall := listNodesCall
+
+			modules := []string{"onload", "sfc"}
+			for _ = range modules {
```

Comment:
```suggestion
			for range modules {
```
Since you're not using either the index or the value, you can skip the dummy definition

Reply:
Neat.